.github/workflows/amplify_integration_tests.yaml

```diff
@@ -130,7 +130,7 @@ jobs:
       - name: Run integration tests
         run: |
           chromedriver --port=4444 &
-          melos exec -c 1 --scope ${{ matrix.scope }} -- flutter drive --driver=test_driver/integration_test.dart --target=integration_test/main_test.dart -d web-server
+          melos exec -c 1 --scope ${{ matrix.scope }} -- \$MELOS_ROOT_PATH/build-support/integ_test.sh -d web-server --retries 1
```

Comment:
Is this considered temporary? I understand retries are necessary for platforms where flakiness is out of our control (emulators not starting for example), but if this issue might be in our control I would rather see it fail.

I'm fine with this for now, but wondering if we should add a task to investigate more.

Reply:
I think a task to investigate more is appropriate, and we could try turning off retries on all non-android platforms to see how noisy it gets.  However, I do think there is some flakiness outside our control that will come and go with new versions of flutter ,etc...

While letting it fail might make sense and raise visibility of errors that need attention, my experience has always been that despite best intentions, people tend to retry failed tests manually without always investigating, so I would prefer to go the direction of logging some metrics on failed tests and retries so we can look for trends while reducing ops load of retrying tests manually.